llvm/lib/Target/AMDGPU/GCNProcessors.td

```diff
@@ -9,11 +9,11 @@
 // The code produced for "generic" is only useful for tests and cannot
 // reasonably be expected to execute on any particular target.
 def : ProcessorModel<"generic", NoSchedModel,
-  [FeatureGDS, FeatureGWS]
+  [FeatureGDS]
```

Comment:
As a side note I'd remove one last feature. IMO generic shall be featureless.

Reply:
I'd even assert somewhere that feature mask for generic is zero.

Reply:
No tests should be removed. The tests should be moved to use an explicit target

Reply:
Tahiti is not the one to support it. No luck here. Technically it has it, but practically it does not.

Reply:
You can ignore FeatureFlatAddressSpace, it is irrelevant now. Just leave it alone